### ISSUE TITLE
void selector is whitespace-sensitive

### ISSUE THREAD

**Issue by gtebbutt** [cgrand/enlive#94]
Possibly expected behaviour, but it seemed worth mentioning: the void selector currently excludes nodes which contain only whitespace (where a CSS :empty selector would match).

``` clojure
[[:div#menu void]]
```

matches: 

``` html
<div id="menu"></div>
```

but not:

``` html
    <div id="menu">
    </div>
```

The fix is very simple: 

``` clojure
(def void (pred #(empty? (remove empty? (map clojure.string/trim (:content %))))))
```

**Comment by justiniac** [comment]
Good point! SGML states that these two snippets are equivalent. Nevertheless your fix may break existing programs that coped with this "feature", so we'll consider it for a major version.
Thankies!

**Comment by cgrand** [comment]
What about adding a dedicated `whitespace` ou `whitespace-only` selector?
